scripts/core/editor3/components/Editor3InitializeSpellchecker.tsx

```diff
@@ -58,7 +58,8 @@ export class Editor3InitializeSpellchecker extends React.PureComponent<IProps, I
                         this.props.dispatch(setExternalOptions({
                             spellchecking: getInitialSpellcheckerData(spellcheck, language),
                         }));
-
+                    }).finally(() => {
```

Comment:
Where does it crash? If language is missing wouldn't be better to simply not call `this.getSpellcheckerStatus` and return?